CRM/Extension/ClassLoader.php

```diff
@@ -179,7 +179,12 @@ private static function loadExtension(\Composer\Autoload\ClassLoader $loader, CR
    * @return string
    */
   protected function getCacheFile() {
-    $envId = \CRM_Core_Config_Runtime::getId();
+    $envId = md5(implode(',', array_merge(
+      [\CRM_Core_Config_Runtime::getId()],
+      array_column($this->mapper->getActiveModuleFiles(), 'prefix')
```

Comment:
Note that in terms of the tangential issue I hit - ie https://github.com/totten/civix/issues/287 - that `getActiveModuleFiles` is loading from the possibly-last-build-relevant `Redis` - I'm trying to figure out the impact of this on that issue. I suspect it would be unchanged

Reply:
Off the top of my head -- I wouldn't expect an effect one way or the other.

In theory, if all the Redis caches had a similar prefix (eg `dmaster/{$envAndModuleId}/{$cacheGroup}/{$cacheKey}`), then that might mitigate the issue. But for the use-case of `civibuild install` or `civibuild reinstall`, I might lean toward updating the installer -- it should pre-emptively clear caches to ensure that the new install starts from a clean slate.